client/public/font/scss/_extras.scss

```diff
@@ -0,0 +1,44 @@
+/* EXTRAS
```

Comment:
Interesting that you used both LESS and SCSS what was the reasoning behind this choice?